src/SFA.DAS.PR.Application/AccountProviders/Queries/GetAccountProviders/GetAccountProvidersQueryValidator.cs

```diff
@@ -0,0 +1,12 @@
+﻿using FluentValidation;
+
+namespace SFA.DAS.PR.Application.AccountProviders.Queries.GetAccountProviders
+{
+    public class GetAccountProvidersQueryValidator : AbstractValidator<GetAccountProvidersQuery>
+    {
+        public GetAccountProvidersQueryValidator()
+        {
+            RuleFor(model => model.AccountId).Must(id => id != 0).WithMessage("Account ID needs to be set.");
```

Comment:
put this message in a constant in the validator, then use that constant in the validators for the test